src/render.ts

```diff
@@ -0,0 +1,96 @@
+import {ReactElement} from 'react';
+import {Ink, createInk, InkOptions} from './ink';
+import {createExperimentalInk} from './experimental/createExperimentalInk';
+import instances from './instances';
+import {ExperimentalDOMNode} from './experimental/dom';
+import {DOMNode} from './dom';
+import {Stream} from 'stream';
+
+export interface RenderOptions {
+	stdout?: NodeJS.WriteStream;
+	stdin?: NodeJS.ReadStream;
+	debug?: boolean;
+	exitOnCtrlC?: boolean;
```

Comment:
Would be nice to include doc comments from all these properties. You can just copy-paste from the readme.